src/index_503/index.py

```diff
@@ -121,7 +121,7 @@ def _make_index_at_temp_dir(self, temp_dir_path: Path) -> None:
             metadata_name = wheel_file_obj.metadata_name
             projects[metadata_name].append(wheel_file_obj)
             canonical_name_to_metadata_name[canonical_name] = metadata_name
-            os.symlink(wheel_file_symlink_target, target_file)
+            os.link(wheel_path, target_file)
```

Comment:
This ensures even if something else uploads a new wheel we have a link to the old file until the index is rebuilt again